### ISSUE TITLE
ReferenceError: HTTP is not defined

### ISSUE THREAD

**Issue by imorti** [meteorclub/cityforks#2]
Got this error at minute 47 when trying to load. odd.

**Comment by queso** [comment]
You got the error with this application from Git, or from one you are trying to make on your own?

You need to "meteor add http"

**Comment by imorti** [comment]
nah, I had to add http. I think missed/skipped that step in the video. 

Thanks for doing this. Good stuff.

**Comment by queso** [comment]
Sure, always glad to help out.

**Comment by imorti** [comment]
trying to get more people interested in MeteorJS in NYC so this helps.

**Comment by queso** [comment]
There is a good meetup there, from what I hear.

**Comment by imorti** [comment]
didn't know. Will find out. thanks!
